.github/workflows/release.yml

```diff
@@ -31,12 +31,23 @@ jobs:
     name: Release and Publish
     runs-on: ubuntu-latest
     needs: test
+    env:
+      VSCE_TOKEN: ${{secrets.VSCE_TOKEN}}
     steps:
       - uses: actions/checkout@master
+
       - uses: actions/setup-node@v1
         with:
           node-version: ^12.13
-      - name: make.js - build, test, and package
-        run: |
-          yarn install
-          node make.js all
+
+      - name: Yarn install
+        run: yarn install
+
+      - name: Build, lint, test and package
+        # Stuck on v1.0  because of https://github.com/GabrielBB/xvfb-action/issues/10
+        uses: GabrielBB/xvfb-action@v1.0
```

Comment:
@philsturgeon This should ensure e2e test don't fail (as they require some virtual X environment to run)